### ISSUE TITLE
"name" attribute is not parsed in by the http_check.pp script

### ISSUE THREAD

**Issue by bshih-ixxus** [DataDog/puppet-datadog-agent#172]
In "http_check.yaml.example", it states that we can specify a "name" attribute. 
```
# - name: My second service 
#   url: https://another.url.example.com 
```
However, the "name" attribute is not being parsed in "http_check.pp" script.
```
[class datadog_agent::integrations::http_check ( 
  $url       = undef, 
  $username  = undef, ](url)
```

Potentially, this can be fixed by modifying the script and the .erb files as below: 
 
1) https://github.com/DataDog/puppet-datadog-agent/blob/master/manifests/integrations/http_check.pp
```
  class datadog_agent::integrations::http_check ( 
+   $svcname   = undef, 
    $url       = undef, 
```
2) https://github.com/DataDog/puppet-datadog-agent/blob/master/templates/agent-conf.d/http_check.yaml.erb 
```
 
  instances: 
!     -   name: <%= @name %> 
          url: <%= @url %> 
  <% if @timeout -%> 
          timeout: <%= @timeout %> 
--- 1,7 ---- 
  init_config: 
   
  instances: 
!     -   name: <%= @svcname %> 
          url: <%= @url %> 
  <% if @timeout -%> 
          timeout: <%= @timeout %> 
```

**Comment by truthbk** [comment]
@bshih-ixxus this is indeed an issue, would you like to submit a PR? Or you'd rather I do it? I'm fine with either, just let me know. If I don't hear from you in a short while, I'll assume you either haven't seen this, or don't really care so I'd submit a patch myself ;)

Thanks for raising this!

**Comment by truthbk** [comment]
Couldn't resist, created the PR here #186 😄

**Comment by bshih-ixxus** [comment]
@truthbk  Of course I care about it 😄 and thanks for raising the PR.

**Comment by truthbk** [comment]
@bshih-ixxus this was merged into the the `1.7.1` bugfix release of the module we made yesterday. Thanks again!
